internal/app/frontend/frontend_service.go

```diff
@@ -122,6 +122,8 @@ func doCreateBackfill(ctx context.Context, req *pb.CreateBackfillRequest, store
 
 	backfill.Id = xid.New().String()
 	backfill.CreateTime = ptypes.TimestampNow()
+	// Set Generation to 1
```

Comment:
Why this comment?  It's entirely redundant with the line below?

Reply:
That's true updated